app/views/analytics/facilities/graphics.html.erb

```diff
@@ -0,0 +1,134 @@
+<div class="snapshot snapshot-simple">
+  <div class="dashboard">
+    <div class="snapshot-header">
+      <span><%= t('analytics.graphics.facility.report_title', organization_name: @organization.name) %></span>
+      <%= @current_month.strftime('%B %Y') %>
+    </div>
+    <h1><%= @facility.name %></h1>
+    <p style="line-height: 1.3; margin: 0.5em 0 0.5em 0;">
```

Comment:
Nit. Can you move these styles to a CSS class and use it here?

Reply:
Thanks for pointing this out. Fixed these now.